src/middleware.ts

```diff
@@ -7,6 +7,7 @@ import { env } from "@/app/env.mjs";
 const clerkMiddleware = authMiddleware({
   // "/" will be accessible to all users
   publicRoutes: ["/"],
+  ignoredRoutes: ["/((?!api|trpc))(_next.*|.+.[w]+$)", "/api/imageInput"],
```

Comment:
no need to ignore any route.
don't even need to interact with middleware.ts

Reply:
Resolved it